src/types/Post.ts

```diff
@@ -0,0 +1,11 @@
+import { Comment } from './Comment';
+import { User } from './User';
+
+export interface Post {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  user: User | null;
```

Comment:
It's better to set the 'User' type as non-nullable, since the task description asks to prepare posts by adding a user and comments to each post. Change the type to 'User' instead of 'User | null'.